.gitignore

```diff
@@ -86,4 +86,4 @@ sklearn/utils/_weight_vector.pyx
 sklearn/utils/_weight_vector.pxd
 sklearn/linear_model/_sag_fast.pyx
 sklearn/metrics/_dist_metrics.pyx
-sklearn/metrics/_dist_metrics.pxd
+sklearn/metrics/_dist_metrics.pxd
```

Comment:
@ShisuiUzumaki This change. You can undo all changes to a file by checking it out from main, e.g. `git checkout main .gitignore`

Reply:
I will do that, but I fail to see the difference between these two line, if you could highlight a bit more

Reply:
It's the deletion of a newline character -- it renders when viewing through the [changed files](https://github.com/scikit-learn/scikit-learn/pull/23852/files) tab. It's very subtle :)

Reply:
ok done

Reply:
@Micy774 if there are no more conflicts, could you merge the changes

Reply:
@ShisuiUzumaki You still need to wait for the review/approval of the core-devs. Usually we require two, but for small changes like this sometimes one is enough. Either way, I'm not a core dev and can't merge your code unilaterally :)